drivers/volume/portworx/portworx.go

```diff
@@ -537,6 +537,8 @@ func (p *portworx) constructStorkVolume(vol *api.Volume) *storkvolume.Info {
 
 	info.AttachedOn = vol.AttachedOn
 
+	info.DirectAttached = p.volumeIsDirectAttached(info)
```

Comment:
nit..It would be better to call this DirectAccess instead of DirectAttached. That way someone looking at the param will know for sure.

Reply:
Same with the function names where we are using Attached as a replacement for Access

Reply:
Done